test/integration_test.rb

```diff
@@ -78,8 +79,118 @@ def test_avoids_bundler_version_if_local_bin_is_in_path
     end
   end
 
+  def test_launch_mode_with_no_gemfile
+    skip("CI only") unless ENV["CI"]
```

Comment:
Unfortunately, these tests print all of the Bundler progress to the screen. However, we can't use `capture_subprocess_io`, because that will mess up with reading from `stdin`.

They are a bit slow, but the main issue is the printing.